docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Monitor
+description: Monitor TempoStack instances
+menuTitle: Monitor
+weight: 300
+aliases:
+- /docs/tempo/operator/monitor
+---
+
+# Monitor
+
+Tempo operator and `TempoStack` operands can be monitored.
+The monitoring configuration for operands is exposed the `TempoStack` CR:
+
+```yaml
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+spec:
+  observability:
+    metrics:
+      createServiceMonitors: true
+      createPrometheusRules: true
+    tracing:
+      sampling_fraction: 1.0
+      jaeger_agent_endpoint: localhost:6831
+```
+
+## Distributed Tracing
+
+All Tempo components as well as the [Tempo Gateway](https://github.com/observatorium/api) support the export of traces in `thrift_compact` format.
+
+### Configure distributed tracing of operands
+
+#### Deploy OpenTelemetry collector
+
+* Deploy [OpenTelemetry Operator](https://opentelemetry.io/docs/k8s-operator/#getting-started) installation.
+* Create an `OpenTelemetryCollector` CR that receives trace data in Jaeger Thrift format and exports data via OTLP to the desired trace backend.
+* *Optional:* Deploy tracing backend to store trace data.
```

Comment:
Adding an opening phrase to provide an introduction to the steps. Also, changed the bullets to steps since this seems like a procedure. 

```suggestion
To deploy the OpenTelemetry collector, follow these steps: 

1. Deploy [OpenTelemetry Operator](https://opentelemetry.io/docs/k8s-operator/#getting-started) installation.
1. Create an `OpenTelemetryCollector` CR that receives trace data in Jaeger Thrift format and exports data via OTLP to the desired trace backend.
1. **Optional:** Deploy tracing backend to store trace data.
```